docs/registry/index.rst

```diff
@@ -276,29 +276,190 @@ For an example of how this might look like, see `GAVO's plate tutorial`_
 
 .. _GAVO's plate tutorial: http://docs.g-vo.org/gavo_plates.pdf
 
+More examples
+-------------
+
+Discover archives
+^^^^^^^^^^^^^^^^^
+
+You can use the registry ``search`` method (or the ``regsearch`` function) 
+to discover archives that may have x-ray images and then query those archives 
+to find what x-ray images that have of CasA. For the arguments you will 
+enter ``'image'`` for the service type and ``'x-ray'`` for the waveband. 
+The position is provided by the Astropy library.
+
+The query returns a :py:class:`~pyvo.registry.regtap.RegistryResults` object 
+which is a container holding a table of matching services. In this example 
+it returns 33 matching services.
+
+.. doctest-remote-data::
+
+  >>> import pyvo as vo
+  >>> from astropy.coordinates import SkyCoord
+  >>>
+  >>> import warnings
+  >>> warnings.filterwarnings('ignore', module="astropy.io.votable.*")
+  >>>
+  >>> archives = vo.regsearch(servicetype='image', waveband='x-ray') 
+  >>> pos = SkyCoord.from_name('Cas A')
+  >>> len(archives)   # doctest: +IGNORE_OUTPUT
+  33
+
+There are also other type of services that you can choose via the 
+``servicetype`` parameter:
+
+  ===== ======================================
+  sia   Simple Image Access (SIA) services
+  ssa   Simple Spectral Access (SSA) services
```

Comment:
Even if the aliases are dismissed from this list, do add `sia2` to this table. 
or maybe, instead link to the class docs ( as `~pyvo.registry.Servicetype`) that has the list (e.g. eliminate the need to maintain such a list at two places).